.github/workflows/deploy-website-next.yml

```diff
@@ -0,0 +1,77 @@
+name: Finish deploying to playground.wordpress.net (don't use manually)
+# Sorry, the previous job just built it and uploaded it to GitHub
+
+on:
+    workflow_run:
+        workflows: [Deploy to playground.wordpress.net]
+        types:
+            - completed
+
+jobs:
+    deploy_to_wp_cloud:
+        # Only run this workflow from the trunk branch and when it's triggered by another workflow OR certain maintainers
+        if: >
+            github.ref == 'refs/heads/trunk' && (
```

Comment:
I thought we could identify this in the `on:` section, and then the entire thing would only run on `trunk` or on merges into `trunk`.

with the `if:` I thought that the job will still run but fail. maybe I'm remembering backwards

Reply:
> with the if: I thought that the job will still run but fail. maybe I'm remembering backwards

@dmsnell it sounds like this will just skip the specific job if the condition is not satisfied:
https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#example-only-run-job-for-specific-repository

We also have a condition set on the `playground-wordpress-net-wp-cloud` workflow environment that says it only runs on "Protected Branches", and in our case that means it only runs on "trunk".

> I thought we could identify this in the on: section, and then the entire thing would only run on trunk or on merges into trunk.

It looks like it is also possible to filter on branch under the `on:`, but we don't use that [in deploy-website.yml](https://github.com/WordPress/wordpress-playground/blob/3b74b4c31534b59f122ef36641aa7daeaca2f905/.github/workflows/deploy-website.yml#L4) which is what this new workflow was adapted from.